src/include/common/keyword/rdf_keyword.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <string>
+
+namespace kuzu {
+namespace common {
+namespace rdf::keyword {
```

Comment:
The nested namespace is not a common practice in the codebase. Why are we making these keyword constants special that they need to be under four-level nested namespaces? I don't have a clue on this.